src/Microsoft.Extensions.Caching.Memory/MemoryCacheOptions.cs

```diff
@@ -9,10 +9,41 @@ namespace Microsoft.Extensions.Caching.Memory
 {
     public class MemoryCacheOptions : IOptions<MemoryCacheOptions>
     {
+        private long? _sizeLimit;
+        private double _removalPercentageOnOvercapacityCompaction = 0.05;
+
         public ISystemClock Clock { get; set; }
 
         public TimeSpan ExpirationScanFrequency { get; set; } = TimeSpan.FromMinutes(1);
 
+        public long? SizeLimit
```

Comment:
I don't want to do a back and forth but forcing users to calculate a percentage seems bizarre.